Makefile

```diff
@@ -41,6 +41,9 @@ include mk/submodule.mk
 # Board specific compiler flags
 include mk/board.mk
 
+# Generate document
+include mk/doc.mk
```

Comment:
Rename it to `docgen` or `doxygen` to avoid implications.

Reply:
Rename it as `docgen.mk`.